### PR TITLE
Feature/new material icons

### LINKED ISSUE #1803: icons should be available in filled and outlined version
- [x] Change rounded icons to filled (must agree with designers)
- [x] implement all icons in a outlined version if available

### PR DESCRIPTION
**Pull request by 01231**
Fixes #1803

# Done is when (DoD):
- [x] Modifications within components are reflected by tests.
- [x] A self-review of the code has been performed.
- [x] The modified component properties have been added to the typescript typings.
- [ ] Potential design changes have been reviewed by a designer. (@Ken)
- [x] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [x] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [x] The modifications are shortly added to CHANGELOG.md with the issue number.
- [x] The modifications still work in IE.
- [x] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/components/00-materials/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 7.0.1
```
Comment (danyball) [inline src/components/00-materials/CHANGELOG.md]: 7.1.0 - its a new "feature"

File: src/components/00-materials/icons-raw/download_outlined.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M13 5v6h1.17L12 13.17 9.83 11H11V5h2m2-2H9v6H5l7 7 7-7h-4V3zm4 15H5v2h14v-2z" fill="currentColor"/></svg>
```
Comment (danyball) [inline src/components/00-materials/icons-raw/download_outlined.svg:1]: `fill="none"` sure?

**Review by danyball — CHANGES_REQUESTED** [review]
Did you forget to run that script, that creates the *.svg.js files? Add them to git, too.

File: src/components/00-materials/icons-raw/download_outlined.svg (hunk 2 above)
Comment (01231) [reply src/components/00-materials/icons-raw/download_outlined.svg:1]: Yes, must be. I had it on `fill="currentColor"` and removed it completely. This is the only way it works.

**Comment by 01231** [comment]
> Did you forget to run that script, that creates the *.svg.js files? Add them to git, too.

I did run it. The files are alle ignored in the `.gitignore`

**Comment by danyball** [comment]
> > Did you forget to run that script, that creates the *.svg.js files? Add them to git, too.
> 
> I did run it. The files are alle ignored in the `.gitignore`

Sorry - you are right....

**Review by danyball — DISMISSED** [review]

File: src/components/00-materials/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+## 8.0.0
+
+- BREAKING CHANGE: `phone` is now `phone_outlined`; `info-flat` and `plus` were deleted
```
Comment (raphaellueckl) [inline src/components/00-materials/CHANGELOG.md:3]: You merged 2 breaking changes into one

File: src/components/00-materials/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+- Updated: `check-circle`, `clear`, `date-input`, `add`, `phone` and `info`
```
Comment (raphaellueckl) [inline src/components/00-materials/CHANGELOG.md:4]: Aren't those breaking changes too? If the size or anything that could break the UI for the user changes, I would highly suggest to als name those as breaking change.

**Review by raphaellueckl — CHANGES_REQUESTED** [review]

File: src/components/00-materials/CHANGELOG.md (hunk 4 above)
Comment (01231) [reply src/components/00-materials/CHANGELOG.md:4]: Size are all the same. This is not a breaking change in my opinion.

**Review by danyball — APPROVED** [review]
lgtm

File: src/components/00-materials/CHANGELOG.md (hunk 3 above)
Comment (01231) [reply src/components/00-materials/CHANGELOG.md:3]: a matter of definitions :)

**Review by raphaellueckl — APPROVED** [review]

**Pull request merged**